### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by teguhsatria892** [JoBinsJP/vue3-datatable#21]
where is the documentation? please provide it

**Comment by eeyvee-0x4d** [comment]
I agree. This really needs documentation.

**Comment by bhojkamal** [comment]
Yes, one page documentation will lift the usage of this package.

**Comment by shershen08** [comment]
added some of the cases in README in this PR - https://github.com/JoBinsJP/vue3-datatable/pull/28
